src/components/text-editor/text-editor.tsx

```diff
@@ -140,6 +140,8 @@ export class TextEditor implements FormComponent<string> {
                 onChange={this.handleChange}
                 value={this.value}
                 aria-controls={this.helperTextId}
+                tabindex={this.disabled ? -1 : 0}
+                aria-disabled={this.disabled}
```

Comment:
these, together with the newly added styles ☝️ 
```scss
    limel-prosemirror-adapter {
        pointer-events: none;
    }
```
will prevent the ProseMirror to become ever focusable using the keyboard or mouse. 

While individual buttons in `limel-action-bar` can accept a `disabled` prop, I don't think we need to do more than this and add that prop to them. I think the screen-readers will also take care of the rest, when they see `aria-disabled`.

Reply:
agree 👍